app/models/request.rb

```diff
@@ -6,7 +6,7 @@ class Request < ApplicationRecord
   has_many :event_requests
   # we model this as many-to-many to allow for future feature enhancements if
   # needed. Initial intention is only one event request per request.
-  has_many :events, through: :event_requests
+  has_many :recurring_events, through: :event_requests
```

Comment:
@hackmastera I'm guessing this change is what is causing the test to fail.  Was this intended?

Reply:
@carolyncole this change was necessary since we renamed `Event` to `RecurringEvent` and then got rid of the `EventInstance` and put its fields on `EventRequest`. I guess it slipped through the cracks in an earlier commit.